content/docs/enterprise/configure.mdx

```diff
@@ -43,7 +43,7 @@ All values are case sensitive unless otherwise noted.
 | <a className="entRef-anchor" id="databroker-service-url">#</a><a href='#databroker-service-url'>DATABROKER_SERVICE_URL</a> | The databroker service URL. | `http://localhost:5443` |
 | <a className="entRef-anchor" id="debug-config-dump">#</a><a href='#debug-config-dump'>DEBUG_CONFIG_DUMP</a> | Dumps the Databroker configuration. This is a debug option to be used only when specified by Pomerium Support. | `false` |
 | <a className="entRef-anchor" id="disable-remote-diagnostics">#</a><a href='#disable-remote-diagnostics'>DISABLE_REMOTE_DIAGNOSTICS</a> | Disable remote diagnostics. | `true` |
-| <a className="entRef-anchor" id="disable-validation">#</a><a href='#disable-validation'>DISABLE_VALIDATION</a> | Disable config validation. | `false` |
+| <a className="entRef-anchor" id="disable-validation">#</a><a href='#disable-validation'>DISABLE_VALIDATION</a> | (deprecated) Disable config validation. | `false` |
```

Comment:
I think it would be nice to mention the replacement here, maybe something like this?

```suggestion
| <a className="entRef-anchor" id="disable-validation">#</a><a href='#disable-validation'>DISABLE_VALIDATION</a> | (deprecated, please update your configuration to set `VALIDATION_MODE=none` instead) Disable config validation.  | `false` |
```

(and similarly, below)

Reply:
I originally tried hyperlinking to the new setting in the parenthetical, but it was a bit busy, like:
`(deprecated; see validation_mode)`.

I do think it needs to direct users to the fix, though, and it will look "busy" either way.  Your suggestion also tells them how to achieve the same effect with the new option, so let's add it.